src/documentation/Email/Email.stories.mdx

```diff
@@ -13,6 +13,8 @@ import Button from '../../components/Button';
   }}
 />
 
+<hr/>
```

Comment:
Can we automate the insertion of the `<hr/>`?

Reply:
sure, will look into that